hugo/content/sponsors/resource-center/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "DORA Sponsor Resource Center"
+date: 2024-04-26T20:58:12-04:00
+draft: false
+bannerTitle: DORA Sponsor Resource Center
+bannerSubtitle: Thank you for sponsoring the 2024 DORA Report
+---
+
+The [2024 DORA survey](https://google.qualtrics.com/jfe/form/SV_8uCHA4aRzcGDjg2?source=sponsor) is now open and we appreciate your efforts to help drive global participation in this year's survey.
+
+## Launch your campaign!
+
+Each sponsor has agreed to launch a campaign with the exclusive purpose of promoting participation in the survey. And to send at least two follow-up messages as a part of that campaign.
+
+Our experience also shows us that the most effective driver of survey participation is direct contact with potential participants. This is why we specifically request that you leverage your contact database for the purposes of a campaign promoting participation in the survey. 
+
+## Why should your customers and community take the survey?
+
+There are lots of reasons your customers and community should participate in this year's survey.  Participation helps all of us hear from additional voices and take even more experiences into consideration as we identify the practices and capabilities that drive technology performance. Additionally, participation provides teams a moment to reflect on how they're doing today.
+
+## Assets for reuse
+
+You are welcome to take inspiration and copy from our [blog post announcing the survey](https://cloud.google.com/blog/products/devops-sre/2024-dora-survey-now-open) as you craft a message that is suitable for your own audience.
+
+Additionally, we have made a number of [DORA Survey Graphics](https://storage.googleapis.com/dora-sponsor-resources/DORA-Survey-Graphics.zip) <small>(ZIP, 175MB)</small> available for download and use.
+
+## Sponsor distribution list
+
+Announcements and updates will be sent to the `dora-sponsors-2024@` distribution list.  Reach out to [mailto:sponsor-dora@google.com](sponsor-dora@google.com) with the names and email addresses of anyone else from your organization who should be added to this distribution list.
```

Comment:
For this link, the `mailto:` prefix should be in the parentheses, not in the brackets.

Reply:
Fixed in with 2121850dcec271f6ca27d647529633ea7c2db8b6